gnucash/gnome/gnc-plugin-page-register.c

```diff
@@ -742,6 +742,10 @@ gnc_plugin_page_register_new_common (GNCLedgerDisplay* ledger)
     if (!gnc_features_check_used (gnc_get_current_book(), GNC_FEATURE_REG_SORT_FILTER))
         gnc_features_set_used (gnc_get_current_book(), GNC_FEATURE_REG_SORT_FILTER);
 
+    // added for version 4.3 onwards
```

Comment:
Shouldn't that be 4.5 (because of the snap release of 4.4)?

Reply:
or is it the result of https://bugs.gnucash.org/show_bug.cgi?id=798070#c4?

Reply:
> Shouldn't that be 4.5 (because of the snap release of 4.4)?

yes, this need to be updated after fixing the issue mentioned at https://bugs.gnucash.org/show_bug.cgi?id=798070#c4? See https://github.com/Gnucash/gnucash/pull/856 for an associated mr.